src/main.rs

```diff
@@ -245,26 +245,39 @@ async fn main() -> Result<()> {
     } else {
         if config.cachegrind {
             let command = "valgrind";
-            let mut args = vec!["--tool=cachegrind".to_owned()];
+            let mut args = vec![
+                "--tool=cachegrind".to_owned(),
+                "--trace-children=yes".to_owned(),
+                // Set some reasonable L1 and LL values. It is important that these
+                // values are consistent across runs, instead of the default.
+                "--I1=32768,8,64".to_owned(),
+                "--D1=32768,8,64".to_owned(),
+                "--LL=8388608,16,64".to_owned(),
+            ];
             args.append(&mut config.run.clone());
             let output = Command::new(command)
                 .args(args)
                 .envs(&config.env)
                 .output()
                 .await?;
             let stderr = String::from_utf8_lossy(&output.stderr);
-            let instructions: f64 = stderr
-                .trim()
-                .lines()
-                .filter(|x| x.contains("I   refs:"))
-                .next()
-                .expect("bad cachegrind output")
-                .trim()
-                .split_whitespace()
-                .last()
-                .expect("bad cachegrind output")
-                .replace(",", "")
-                .parse()?;
+
+            let lines = stderr.trim().lines().filter(|x| x.contains("I   refs:"));
+            let mut instructions: f64 = 0.0;
+            for line in lines {
```

Comment:
Do we expect more than one line to contain `I   refs:`?

Reply:
yeah, we'll get one result per-child process

Reply:
ahhh gotcha. sgtm.